api/src/test/java/org/openmrs/module/attachments/obs/ImageAttachmentHandlerIT.java

```diff
@@ -22,7 +24,7 @@
 	protected TestHelper testHelper;
 
 	@Test
-	public void saveComplexData_shouldSaveThumbnailToDisk() throws IOException {
```

Comment:
@oslynn please leave the existing test untouched. However you will need to bring new test cases that are specific to the case of small images.

**Test case 1**: `saveComplexData_shouldNotSaveThumbnailWhenSmallImage`
This test case will replay the saving of an image, but a _small_ image whose original dimensions are under 200 x 200. And it will have to verify that when saving such image, no new thumbnail is generated and the original image file is renamed. 

**Test case 2**: `readComplexData_shouldAlwaysFetchOriginalImageWhenSmallImage`
We want to check that we fetch the original image, whether we ask for the thumbnail or not, when we are in the case of a small image.

You will need `TestHelper` to cover that case with a new, small resource. I can do that for you in an upcoming commit of mine.